packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -55,6 +56,11 @@ enum SelectionChangedCause {
   /// Keyboard-triggered selection changes may be caused by the IME as well as
   /// by accessibility tools (e.g. TalkBack on Android).
   keyboard,
+
+  /// The user used the mouse to change the selection by dragging over a piece
+  /// of text.
+  /// [QUESTION]: Should we call it `drag` instead of `mouse`?
```

Comment:
Don't forget to remove this "[QUESTION]" line.

Reply:
Are we still treating a quick click as a tap?

Reply:
@xster yes.

Reply:
Also, prefer using `//` for comments like that. This way even if you accidentally submit it it won't show up in dartdocs.